File: submissions/example-santa/tree.swift
func printTree(_ height: Int) {
    for i in 0..<height {
        let spaces = String(repeating: " ", count: height - i - 1)
        let stars = String(repeating: "*", count: 2 * i + 1)
        print(spaces + stars)
    }
    let trunkSpaces = String(repeating: " ", count: height - 1)
    print(trunkSpaces + "|")
}

printTree(5)
print("\nMerry Christmas! 🎄")

File: submissions/example-santa/Tree.java
public class Tree {
    public static void main(String[] args) {
        int height = 5;
        for (int i = 0; i < height; i++) {
            for (int j = 0; j < height - i - 1; j++) System.out.print(" ");
            for (int k = 0; k < 2 * i + 1; k++) System.out.print("*");
            System.out.println();
        }
        for (int i = 0; i < height - 1; i++) System.out.print(" ");
        System.out.println("|");
        
        System.out.println("\nMerry Christmas! 🎄");
    }
}
